tests/api/getResolvedName.php

```diff
@@ -82,7 +82,7 @@ public function dataProvider() {
      */
     public function testGetResolvedName($contents, $expectedName) {
         $node = $this->getNodeAtPosition($contents);
-        $this->assertTrue($node instanceof Node\QualifiedName, "Node is not QualifiedName: " . get_class($node));
+        $this->assertInstanceOf(Node\QualifiedName::class, $node, "Node is not QualifiedName: " . get_class($node));
```

Comment:
I assume the custom message is not needed anymore?

Reply:
Gonna remove it soo.